content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,374 @@
+
+Rewards are paid for reporting a slashable _fault_ at epoch end. The _offending_ validator's share of the epoch's staking rewards is forfeited and paid to the  _reporting_ validator `treasury` account for distribution along with the staking rewards at epoch end.
+
+### Accountability prerequisites 
+
+To participate in AFD a [validator](/glossary/#validator) must be a [consensus committee](/glossary/#consensus-committee) member.
+
+## Accountability and Fault Detection protocol
+
+AFD roles, core concepts, and the lifecycle of accountability event processing from accusation to slashing.
+
+### Roles
+
+As consensus committee members, validators play the following roles in AFD processing:
+
+- _reporting_: as the validator reporting a (suspected) rule infraction and submitting new _fault proof_ / _accusation_ on-chain
+- _offending_: as the validator committing a suspected rule infraction and being requested to submit _innocence_ proofs on-chain.
+- _committee member_: as a validator in the consensus committee executing Autonity's consensus protocol and for AFD handling and processing accountability events, maintaining system state for accountability events, and computing and applying slashing penalties.
+
+Validators play the following economic roles in AFD processing:
+
+- _offender_: loss of stake, validator reputation, staking reward revenue as the _offending_ validator of a slashed _fault_
+- _beneficiary_: gain of the _offending_ validator's protocol rewards as the _reporting_ validator of a slashed _fault_.
+
+Autonity community play the following economic role in AFD processing:
+
+- _beneficiary_: gain of slashed stake token for community funds.
+
+### Protocol primitives
+
+Essential primitives of AFD are: accusation, innocence, and fault proofs; slashing and severity; jailing.
+
+#### Accusations
+
+An _accusation_ is a claim that a consensus committee member has failed to participate correctly in consensus. I.e. it has failed to adhere to a rule governing validator participation in consensus and so is guilty of a rule infraction. Protocol only allows a validator to be under _accusation_ once at a time. New _accusations_ are made by submitting evidence of an accusable rule infraction on-chain as an accountability event of type `Accusation`.
+
+Accusations do not automatically cause slashing and an _innocence proof window_ measured in blocks gives the accused _offending_ validator a window to detect an _accusation_ and prove _innocence_ by submitting an _innocence_ proof on-chain.
+
+There are protocol constraints on when an _accusation_ can be made. A _reporting_ validator can only submit an _accusation_ _proof_ if:
+
+- the _offending_ validator:
+  - does not already have a _fault_ in the epoch for which the _new accusation_ is being made for an offence with a higher _severity_
```

Comment:
```suggestion
  - has not been already punished in the epoch for which the new _accusation_ is being made for, for an offence with a higher severity.
```

This sentence was a bit hard to parse for me, not sure if my version is better though.

Reply:
Hmm, take out the `, for` and that's fine I think:

- has not been already punished in the epoch for which the new _accusation_ is being made for an offence with a higher severity.